srcpkgs/git-team/template

```diff
@@ -1,19 +1,16 @@
 # Template file for 'git-team'
 pkgname=git-team
-version=1.5.5
+version=1.6.0
 revision=1
 build_style=go
 go_import_path=github.com/hekmekk/git-team
-go_package="${go_import_path}/cmd/git-team ${go_import_path}/cmd/prepare-commit-msg"
 short_desc="Manage and enhance git commit messages with co-authors"
 maintainer="Frank Steinborn <steinex@nognu.de>"
 license="MIT"
 homepage="https://github.com/hekmekk/git-team"
 distfiles="${homepage}/archive/v${version}.tar.gz"
-checksum=bc88552e78ad67327b153cdc5aeb01ddcb5a99579787a68177fae7aa1a20c27b
+checksum=d474d759b81a458519c47059990d12808df28d4b313a69234aaa9574a70da723
 
 post_install() {
 	vlicense LICENSE
-	mv ${DESTDIR}/usr/bin/prepare-commit-msg ${DESTDIR}/usr/bin/prepare-commit-msg-git-team
-	vinstall bash_completion/git-team.bash 0644 usr/share/bash-completion/completions git-team
```

Comment:
Can you explain these removals in the commit message? (were the files removed? did something change how they work? ...)

Reply:
* The mv isn't necessary anymore, because prepare-commit-msg isn't shipped anymore as a seperate script, instead it get's created on demand in `~/.git-team/hooks` by the main git-team binary.
* The bash completion file indeed is gone, instead there is the `git-team completion` command now which one can use in their bash startup scripts to source completion for git-team.

Reply:
Can you just copy your comment comment into the commit message, then?

Reply:
reworded a bit but done.